tekton/src/pipelines/fvt-assist.yml.j2

```diff
@@ -78,6 +78,18 @@ spec:
         - input: "$(params.mas_app_channel_assist)"
           operator: notin
           values: [""]
+    - name: imagescan
```

Comment:
the name needs to be seet to `imagescan-{appId}`, e.g. imagescan-assist in this case .. same applies to all the other files as well